modules/virt-vm-storage-volume-types.adoc

```diff
@@ -30,6 +30,10 @@ PVC.
 DataVolumes build on the *persistentVolumeClaim* disk type by managing the process
 of preparing the virtual machine disk via an import, clone, or upload operation.
 VMs that use this volume type are guaranteed not to start until the volume is ready.
++
+Define DataVolumes with a `type` of *dataVolume*. If another volume type is
+used in error, such as *PersistentVolumeClaim*, a warning will be triggered
```

Comment:
I'd make this sentence use present instead of future tense, per the IBM Style Guide:

> Use past or future tense only when you cannot use present tense or it does not make sense to use
present tense.

Reply:
Revised.